compiler/compile/OMRSymbolReferenceTable.hpp

```diff
@@ -803,6 +804,10 @@ class SymbolReferenceTable
    TR::SymbolReference * findOrCreateTemporaryWithKnowObjectIndex(TR::ResolvedMethodSymbol * owningMethodSymbol, TR::KnownObjectTable::Index knownObjectIndex);
    TR::SymbolReference * findOrCreateThisRangeExtensionSymRef(TR::ResolvedMethodSymbol *owningMethodSymbol = 0);
    TR::SymbolReference * findOrCreateContiguousArraySizeSymbolRef();
+#if defined(J9VM_GC_ENABLE_SPARSE_HEAP_ALLOCATION)
+   TR::SymbolReference * findOrCreateContiguousArrayDataAddrFieldShadowSymRef();
+#endif // defined(J9VM_GC_ENABLE_SPARSE_HEAP_ALLOCATION)
+   TR::SymbolReference * findContiguousArrayDataAddrFieldShadowSymRef();
```

Comment:
Why is the `findOrCreate` inside an ifdef but the `find` routine is not ?

Reply:
Sorry about the delay, I must have missed the email for comment. But `findOrCreate` routine uses `TR::Compiler->om.offsetOfContiguousDataAddrField` API which needs to be guarded by ifdef. However, `find` isn't guarded because it is used by `node->isDataAddrPointer()`. I left `find` outside to avoid having ifdef in `node->isDataAddrPointer()`.